cmd/ftl/cmd_ps.go

```diff
@@ -27,7 +27,7 @@ func (s *psCmd) Run(ctx context.Context, client ftlv1connect.ControllerServiceCl
 		return errors.WithStack(err)
 	}
 	if s.JSON {
-		marshaller := jsonpb.Marshaler{}
+		marshaller := jsonpb.Marshaler{Indent: "  "}
```

Comment:
This one makes sense, but the other two shouldn't be indented, as they're part of a columnar output and indenting them will just corrupt the columns.

Reply:
ah cool, fixed